scripts/packages/chocolatey/bazel.nuspec.template

```diff
@@ -75,8 +75,9 @@ Supply like `--params="/option:value ..."` ([see docs for --params](https://gith
       <dependency id="chocolatey-uninstall.extension" />
     </dependencies>-->
     <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.0.7"/>
       <dependency id="jdk8" version="[8.0.102,)"/>
-      <dependency id="msys2" version="[20160719.1,)"/>
+      <dependency id="msys2" version="[20160719.1.0]"/>
```

Comment:
Ah :) I've seen failures with newer major versions (20170130 specifically), but I also haven't tested with 20160719's other minor/patch versions. The problem was binary incompatibility with msys-2.0.dll between the two major versions, but I believe they should be compatible within the same major one.

Aaand fortunately this will soon not matter because we're close to removing the msys dependency :)  (we'll still need some msys installed in order to have a bash.exe to run, but bazel itself won't be tied to a given version).